sites/uat/features/high-value/stanford_sites_helper.feature

```diff
@@ -5,33 +5,27 @@ Feature: Stanford Sites Helper Module
   I want to ensure the Stanford Sites Helper module is working properly
 
   @api @safe @deploy @javascript
-  Scenario Outline: Homepage text content
```

Comment:
Hmmm... interesting. Scenario Outlines are slow. Is this noted anywhere out there in the interwebs?